.ci/setup.sh

```diff
@@ -40,6 +40,13 @@ fi
 echo "Install Kata from sources"
 bash -f ${cidir}/install_kata_from_source.sh
 
+# Install docker after install kata image
```

Comment:
So I think maybe this is not so much 'install docker', as 'install kata into docker', is it? As we now know that docker will always already be installed, as the `install_kata_from_source.sh` would not have worked otherwise?